.github/ISSUE_TEMPLATE/generic_issue.yaml

```diff
@@ -0,0 +1,126 @@
+name: Report a generic issue
```

Comment:
So actual bugs with conversation in HA should be reported in core. Maybe this should just be for sentences? 

We could still have a template picker, where feature requests link to the forums, like this: https://github.com/home-assistant/frontend/blob/dev/.github/ISSUE_TEMPLATE/config.yml